relay-server/src/services/stats.rs

```diff
@@ -194,7 +194,7 @@ impl RelayStats {
 
         metric!(
             gauge(RelayGauges::AsyncPoolQueueSize) = metrics.queue_size(),
-            pool_name = async_pool.name()
+            pool = async_pool.name()
         );
         metric!(
             gauge(RelayGauges::AsyncPoolUtilization) = metrics.utilization() as f64,
```

Comment:
Wanna also take the opportunity to * 100 the metric, then it matches the utilization we calculate for the services?